src/public/index.development.html

```diff
@@ -67,9 +67,9 @@
 
         <link rel="stylesheet" href="<%= htmlWebpackPlugin.files.css[0] %>">
 
-        <link rel="shortcut icon" href="<%= htmlWebpackPlugin.options.publicPath %>media/favicon/favicon.png">
-        <link rel="icon" href="<%= htmlWebpackPlugin.options.publicPath %>media/favicon/favicon.png">
-        <link rel="manifest" href="<%= htmlWebpackPlugin.options.publicPath %>media/webmanifest/manifest.webmanifest">
```

Comment:
Why hardcode this? Isn't it passed from the configs just fine?

Reply:
Currently there was no support for favicon & manifest in configs, and it didn't seam as necessity as all data that is hard coded is static, meaning that file paths always will stay the same + previous implantation also used hardcoded favicon paths.